victoria_sensor_msgs/msg/EncoderCount.msg

```diff
@@ -0,0 +1,5 @@
+# This message represents the count value
+# for an encoder.
+
+std_msgs/Header header
+int64 count
```

Comment:
I would have suggested min but you're use of an `int64` says the min is zero, unless the encoder would reset higher than that, which... I don't **think** makes any sense.